### PR TITLE
Update pyscaffold v3.2.3

### PR DESCRIPTION
**Pull request by sebhahn**
- Package no longer Python 2.7 compatible
- Update pyscaffold v3.2.3
- Remove build_script and changes.md
- Update environment.yml
- Package name changed from pynetCF to pynetcf
- Updating travis settings

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,13 @@ notifications:
   email: false
 python:
   # We don't actually use the Travis Python, but this keeps it organized.
-  - "2.7"
-  - "3.5"
   - "3.6"
```
Comment (cpaulik) [inline .travis.yml:7]: Should we upgrade to 3.7 or 3.8?

File: docs/conf.py [hunk 2]
```diff
@@ -61,7 +83,7 @@
 
 # General information about the project.
 project = u'pynetcf'
-copyright = u'2014, '
+copyright = u'2020, Sebastian'
```
Comment (cpaulik) [inline docs/conf.py:86]: TU Wien

File: docs/conf.py [hunk 3]
```diff
@@ -8,34 +8,56 @@
...
 
-import sys
 import os
+import sys
 import inspect
-from sphinx import apidoc
-
-import mock
-
-MOCK_MODULES = ['numpy', 'pandas', 'netCDF4',
```
Comment (cpaulik) [inline docs/conf.py:18]: I think we still need the mocking for the documentation build.

File: docs/conf.py [hunk 4]
```diff
@@ -110,24 +132,19 @@
...
-    # add package path to python path
-    sys.path.append('..')
-
-if not on_rtd:  # only import and set the theme if we're building docs locally
-    import sphinx_rtd_theme
-    html_theme = 'sphinx_rtd_theme'
-    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+html_theme = 'alabaster'
```
Comment (cpaulik) [inline docs/conf.py:139]: sphinx_rtd_theme?

File: docs/conf.py [hunk 5]
```diff
@@ -220,7 +237,7 @@
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
     ('index', 'user_guide.tex', u'pynetcf Documentation',
-     u'TU Wien', 'manual'),
+     u'Sebastian', 'manual'),
```
Comment (cpaulik) [inline docs/conf.py:240]: TU Wien

File: docs/index.rst [hunk 6]
```diff
@@ -4,15 +4,24 @@ pynetcf
 
 This is the documentation of **pynetcf**.
 
-Basic python classes that map to netCDF files on disk written according
```
Comment (cpaulik) [inline docs/index.rst:7]: I think none of those changes should be taken over from pyscaffold template

File: requirements.txt [hunk 7]
```diff
@@ -1,5 +1,17 @@
-numpy
-netCDF4>=1.4.2
-pygeogrids
-pygeobase>=0.3.18
-pandas!=0.19.0
+# =============================================================================
```
Comment (cpaulik) [inline requirements.txt:1]: These should be repeated in `install_requires` in the `setup.cfg` file. See the instructions created by pyscaffold in this file.

File: .travis.yml (hunk 1 above)
Comment (sebhahn) [reply .travis.yml:7]: I'll add both

**Comment by sebhahn** [comment]
Currently all codes use `import pynetcf` and not `import pynetCF`
Making a new scaffold template with `pynetCF` would have required to change all codes using the package at the moment

**Comment by cpaulik** [comment]
Yeah. Also [PEP8 recommends it](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) so lets do lowercase.

**Comment by sebhahn** [comment]
@cpaulik thanks a lot for your help!

**Pull request merged**